search-api/tests/conftest.py

```diff
@@ -37,13 +39,30 @@ def not_raises(exception):
 
 
 @pytest.fixture(scope='session')
-def app():
+def ld():
+    '''LaunchDarkly TestData source.'''
+    td = TestData.data_source()
+    yield td
+
+
+@pytest.fixture(scope='session')
+def app(ld):
     """Return a session-wide application configured in TEST mode."""
-    _app = create_app('testing')
+    _app = create_app('testing', **{'ld_test_data': ld})
 
     return _app
 
 
+@pytest.fixture
+def set_env(app):
+    '''Factory to set environment and Flask config variables.'''
```

Comment:
Doesn't really matter too much, but we always use double quotes for the method desc

Reply:
yep, will change that, the linter doesn't like it